onnxruntime/core/providers/nuphar/compiler/x86/op_ir_creator/tensor/scatter.cc

```diff
@@ -0,0 +1,51 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#include "core/providers/nuphar/compiler/x86/op_ir_creator/all_ops.h"
+
+#include "core/framework/op_kernel_info.h"
+#include "core/providers/nuphar/mti_x86/tensor/scatter.h"
+
+namespace onnxruntime {
+namespace nuphar {
+
+static Status ScatterCommon(
+    const tvm::Array<tvm::Tensor>& inputs,
+    const Node& node,
+    tvm_codegen::CodeGenContext&,
+    tvm::Array<tvm::Tensor>& outputs,
+    const std::string& name) {
+  ProtoHelperNodeContext ctx(node);
+  OpNodeProtoHelper<ProtoHelperNodeContext> attrs(&ctx);
+
+  // The default value of optional attribute axis is 0
+  int64_t axis;
+  if (!attrs.GetAttr<int64_t>("axis", &axis).IsOK()) {
+    axis = 0;
+  }
```

Comment:
Would be more concise using GetAttrOrDefault

Reply:
Fixed.